netbout-web/src/test/java/com/netbout/rest/TkAppITCase.java

```diff
@@ -80,7 +93,7 @@ public void downloadsStaticResources() throws Exception {
     }
 
     /**
-     * TkApp can render non-found pages.
+     * TkApp can render non-found pages while adding a severe level notice.
```

Comment:
@original-brownbear I would reword it to something like `TkApp can render non-found pages and add flash message with a severe level`